benchmarking/daint_single_node/run_benchmark.py

```diff
@@ -0,0 +1,183 @@
+#!/usr/bin/env python3
+
+# This script generates a rundirectory given user-provided arguments and a configuration
+# file. It relies on fv3config for setting up the run directory. Once done, a SLURM
+# script is generated and submitted. Optionally, we wait for the completion of the SLURM
+# job.
+
+# 2021/01/22 Oliver Fuhrer, Vulcan Inc, oliverf@vulcan.com
+
+import math
+import os
+import sys
+import shutil
+import hashlib
+import yaml
+import click
+import fv3config
+
+# constants
+DAINT_SECTION = 'hybrid' # one of hybrid or multicore
```

Comment:
do we have access to MC?

Reply:
![image](https://user-images.githubusercontent.com/5963462/105757495-def36b00-5f02-11eb-9bd5-2d832c63892d.png)

Reply:
I am highly confused
![Screenshot from 2021-01-25 22-42-57](https://user-images.githubusercontent.com/5988437/105775069-776f0700-5f66-11eb-8660-d245da4c293f.png)